interfaces/payment-handler.idl

```diff
@@ -4,6 +4,7 @@ interface PaymentManager {
     [SameObject]
     readonly attribute PaymentInstruments instruments;
     [Exposed=Window] static Promise<PermissionState> requestPermission();
+    attribute DOMString userHint;
```

Comment:
This probably shouldn't be added yet, unless it's also going to be tested

Reply:
Removed 'userHint'.